backend/app/tests/api/routes/schedule/test_convert_db_schedule_to_json.py

```diff
@@ -0,0 +1,27 @@
+from app.api.routes.schedule import convert_db_schedule_to_json
+from app.models import Schedule, SchedulePublic
+
+
+def test_convert_db_schedule_to_json_should_convert_db_to_serializable_object() -> None:
```

Comment:
Do we want to consider camelcase? Also, I'm not sure the test_ prefix is needed -- we know it's a test, it's in a test file. :)

Reply:
https://peps.python.org/pep-0008/#function-and-variable-names
```
Function names should be lowercase, with words separated by underscores as necessary to improve readability.

Variable names follow the same convention as function names.

mixedCase is allowed only in contexts where that’s already the prevailing style (e.g. threading.py), to retain backwards compatibility.
```

Additionally, it's good to have an quick distinction from our frontend TS styles, which share many of the same names

Reply:
The `test_` prefix also seems to be standard.  More importantly, if you remove this prefix, the tests will not run.  There is probably a way to override this but it would run counter to common practice.

Reply:
Oh interesting. Ok, sounds good.